hnn_core/network.py

```diff
@@ -1428,6 +1431,75 @@ def add_electrode_array(self, name, electrode_pos, *, conductivity=0.3,
                                      method=method,
                                      min_distance=min_distance)})
 
+    def update_weights(self, e_e=None, e_i=None,
+                       i_e=None, i_i=None, copy=True):
```

Comment:
Should the default behavior of the method be copy=False?  Will most use cases be updating the weights in-place or saving to a new object?

Reply:
agreed! will change to false and then good to merge